_util/os.ts

```diff
@@ -17,4 +19,4 @@ export const osType = (() => {
   return "linux";
 })();
 
-export const isWindows = osType === "windows";
+export const isWindows: boolean = osType === "windows";
```

Comment:
This is a trivial type annotation and not required.

Reply:
Exactly, I will revert this line